src/index.html

```diff
@@ -4,9 +4,396 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>  
+      <div class="menu">
+        <a class="logo-link menu__logo-link" href="#">
+          <img 
+            class="logo-link__logo-img" 
+            src="images/logo.png" 
+            alt="logo"
+          >
+        </a>
```

Comment:
why this does not belong to its BEM-block?

Reply:
Because the logo will most likely need to be reused somewhere.